governance.md

```diff
@@ -1,149 +1,285 @@
+    - initial members TBD (historically bgrant0607, thockin, smarterclayton, erictune)
+    - designed and reviewed several APIs in the system
+    - familiar with the design, requirements, mechanics, conventions, style,
+      scope, gotchas, etc. of the API to be in `kubernetes/pkg/api` and `kubernetes/pkg/apis`
+      OWNERS files `approvers` lists
+      - TODO: decouple design review from the firehose of PR reviews
+    - ensure Kubernetes has a consistent [API style](contributors/devel/api-conventions.md),
+      patterns, and philosophies
+    - guide development of new APIs
+    - codify new patterns for novel problems
+  - Expectations
+    - review and approve API changes and proposals in their functional area
+  - Benefits
+    - TBD
+    - added to [`api-approvers`](https://github.com/orgs/kubernetes/teams/api-approvers) github team
 
 ## SIG roles
-- **SIG Participant**: active in one or more areas of the project; wide 
-  variety of roles are represented
-- **SIG Lead**: SIG organizer
+- **SIG PARTICIPANT**: active in one or more areas of the project; wide variety of roles are represented
+- **SIG LEAD**: SIG organizer 
+  - Expectations 
+    - ensure meetings occur, decisions are communicated
+    - facilitate planning for each release and ensure feature process is followed
+    - ensure other SIGs are able to provide input / feedback
+    - ensure SIG follows governance guidelines, below
 
 ## Management roles
-- **Team Lead**: tech lead or manager of some team at some company working on 
-  K8s; can influence priorities of their team members; pragmatically, 
-  probably want label/assignment powers
-- [**kubernetes-pm**](https://github.com/orgs/kubernetes/teams/kubernetes-pm): help to [manage and maintain the project](project-managers/README.md) in
-  ways other than just writing code (e.g. managing issues); should subscribe to kubernetes-pm@googlegroups.com
+- [**PM**](https://github.com/orgs/kubernetes/teams/kubernetes-pm): help to [manage and
+  maintain the project](project-managers/README.md) in ways other than just writing code (e.g. managing
+  issues); owns feature tracking and roadmap planning; should subscribe to kubernetes-pm@googlegroups.com
+  - Requirements, expectations, benefits TBD
+  - Note that the PM SIG is current distinct from the kubernetes-pm github team, described above
 
 ## Rotations
-- [**Build Cop**](contributors/devel/on-call-build-cop.md): ensure tests pass, submit queue is working, rollback PRs, 
-  manually merge as necessary to fix build; should be members of appropriate repo's build-cops github team
-  (e.g., [kubernetes-build-cops](https://github.com/orgs/kubernetes/teams/kubernetes-build-cops))
-- [**User-Support Rotation**](contributors/devel/on-call-user-support.md): answer questions on stackoverflow, googlegroups, 
-  slack, twitter, etc. full time while on duty
+- [**Build Cop**](contributors/devel/on-call-build-cop.md): ensure tests pass, submit queue is working,
```

Comment:
Why not break it out into a seed of a new doc now.  At least point to *something* here so that we can reorganize the contributor docs there.  Bloating this doc with unnecessary things because we have a mess elsewhere doesn't make sense to me.